signers.sh

```diff
@@ -75,12 +75,12 @@ exec_run() {
 }
 
 exec_demo() {
-  cargo run -p signer --bin demo-cli donation --amount 10000
-  cargo run -p signer --bin demo-cli deposit --amount 10000000 # 0.1 BTC
+  cargo run -p signer --bin demo-cli --features "testing" donation --amount 10000
+  cargo run -p signer --bin demo-cli --features "testing" deposit --amount 10000000 # 0.1 BTC
```

Comment:
qq: do we need this? Given that we have `required-features = ["testing"]` in `demo-cli` stanza.

Reply:
Yes, unfortunately I think that we do. I believe the `required-features = ["testing"]` line just says that we need to specify the features for that binary. I think that I'll update the `demo_cli` so that it isn't necessary, because it is a pain to have to specify the feature.

Reply:
Or we can add an option to `signers.sh` to just run `cargo run -p signer --bin demo-cli --features "testing" $@`

Reply:
I changed the demo-cli so that it no longer requires the testing feature in https://github.com/stacks-network/sbtc/pull/1446/commits/56937fa8d94d257c65e84b49b4c35483860423d7